app/models/work-item.ts

```diff
@@ -146,6 +146,18 @@ export async function getNextWorkItem(
 
     await jobQuery;
 
+    // const uquery = tx(Job.table)
```

Comment:
Not sure if this was added for debugging, but looks like it can be removed.

Reply:
Yeah - leftover debugging. Removed.